.infrastructure/cronjob.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: todoapp-cronjob
+  namespace: todoapp
```

Comment:
The namespace is set to `todoapp`, but it should be `mateapp` as per the task requirements. Please update the namespace to avoid deployment issues.